app/waf/src/models/monitor.model.ts

```diff
@@ -99,36 +130,76 @@ export class Monitor extends CommonEntity {
   @property({
     type: 'number',
     default: 500,
+    openapi: {
+      type: 'integer',
+      minimal: '1',
+      maxmium: '65535',
+    },
   })
   adaptiveDivergenceMilliseconds: number;
 
   @property({
     type: 'number',
     default: 100,
+    openapi: {
+      type: 'integer',
+      minimal: '0',
+      maxmium: '100',
+    },
   })
   adaptiveDivergencePercentage: number;
 
   @property({
     type: 'string',
     default: 'relative',
+    openapi: {
```

Comment:
enum: 'absolute' or 'relative'

Reply:
the default value is relative according to the as3 manual.

> enum: 'absolute' or 'relative'

The default value is relative according to the as3 manual.